kitsune/search/config.py

```diff
@@ -5,12 +5,31 @@
 # number of times to retry updates if a conflict happens before raising
 UPDATE_RETRY_ON_CONFLICT = 2
 
+ES_MAPPINGS = [
```

Comment:
why aren't we using synonyms for this case too?

Reply:
discussed in meeting, added a comment